plugins/grab.py

```diff
@@ -103,13 +100,13 @@ def grab(text, nick, chan, db, conn):
             # check to see if the quote has been added
             if check_grabs(name.lower(), msg, chan):
                 return "I already have that quote from {} in the database".format(text)
-                break
```

Comment:
If you remove this break it will continue searching through the conn history and add a quote that is not the last thing someone said.